fe/fe-core/src/main/java/org/apache/doris/tablefunction/MetadataGenerator.java

```diff
@@ -624,22 +627,35 @@ private static TFetchSchemaTableDataResult viewDependencyMetadataResult(TSchemaT
         if (!params.isSetCurrentUserIdent()) {
             return errorResult("current user ident is not set.");
         }
+        List<Expression> conjuncts = Collections.EMPTY_LIST;
+        if (params.isSetFrontendConjuncts()) {
+            conjuncts = FrontendConjunctsUtils.convertToExpression(params.getFrontendConjuncts());
+        }
         Collection<DatabaseIf<? extends TableIf>> allDbs = Env.getCurrentEnv().getInternalCatalog().getAllDbs();
         TFetchSchemaTableDataResult result = new TFetchSchemaTableDataResult();
         List<TRow> dataBatch = Lists.newArrayList();
         ConnectContext ctx = new ConnectContext();
         ctx.setEnv(Env.getCurrentEnv());
         for (DatabaseIf<? extends TableIf> db : allDbs) {
-            List<? extends TableIf> tables = db.getTables();
             String dbName = db.getFullName();
+            if (FrontendConjunctsUtils.isFiltered(conjuncts, "VIEW_SCHEMA", dbName)) {
```

Comment:
should add a conjunct filter to remove the conjunct that do not contain slot named `VIEW_SCHEMA` to avoid do useless filter for each tuple.